sklearn/inspection/_permutation_importance.py

```diff
@@ -157,6 +181,17 @@ def permutation_importance(
 
         .. versionadded:: 0.24
 
+    max_samples : int or float, default=1.0
+        The number of samples to draw from X to compute feature importance
+        in each repeat (without replacement).
+
+        - If int, then draw `max_samples` samples.
+        - If float, then draw `max_samples * X.shape[0]` samples.
+        - If `max_samples` is equal to `1.0` or `X.shape[0]`, all samples
+          will be used.
+
```

Comment:
Maybe extend this parameter docstring a bit to explain why this is useful:

In combination with `n_repeats`, this allows to control the computational speed vs statistical accuracy trade-off of this method.

Reply:
Good point, I took this sentence plus another one.